Mastodon/Scene/Settings/Server Details/AboutInstanceViewController.swift

```diff
@@ -0,0 +1,158 @@
+    func showAdminAccount(_ viewController: AboutInstanceViewController, account: Mastodon.Entity.Account)
+    func sendEmailToAdmin(_ viewController: AboutInstanceViewController, emailAddress: String)
+}
+
+class AboutInstanceViewController: UIViewController {
+
+    weak var delegate: AboutInstanceViewControllerDelegate?
+    var dataSource: AboutInstanceTableViewDataSource?
+
+    let tableView: UITableView
+    let headerView: AboutInstanceTableHeaderView
+    let footerView: AboutInstanceTableFooterView
+
+    var instance: Mastodon.Entity.V2.Instance?
+
+    init() {
+        tableView = UITableView(frame: .zero, style: .insetGrouped)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.register(ContactAdminTableViewCell.self, forCellReuseIdentifier: ContactAdminTableViewCell.reuseIdentifier)
+        tableView.register(AdminTableViewCell.self, forCellReuseIdentifier: AdminTableViewCell.reuseIdentifier)
+
+        headerView = AboutInstanceTableHeaderView()
+        footerView = AboutInstanceTableFooterView()
+
+        super.init(nibName: nil, bundle: nil)
+
+        let dataSource = AboutInstanceTableViewDataSource(tableView: tableView) { tableView, indexPath, itemIdentifier in
+            switch itemIdentifier {
+
+                case .adminAccount(let account):
+                    guard let cell = tableView.dequeueReusableCell(withIdentifier: AdminTableViewCell.reuseIdentifier, for: indexPath) as? AdminTableViewCell else { fatalError("WTF?! Wrong cell.") }
+
+                    cell.condensedUserView.configure(with: account, showFollowers: false)
+
+                    return cell
+
+                case .contactAdmin:
+                    guard let cell = tableView.dequeueReusableCell(withIdentifier: ContactAdminTableViewCell.reuseIdentifier, for: indexPath) as? ContactAdminTableViewCell else { fatalError("WTF?! Wrong cell.") }
+
+                    cell.configure()
+
+                    return cell
+            }
+        }
+
+        tableView.delegate = self
+        tableView.dataSource = dataSource
+
+        self.dataSource = dataSource
+
+        view.addSubview(tableView)
+
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    private func setupConstraints() {
+        let constraints = [
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            view.trailingAnchor.constraint(equalTo: tableView.trailingAnchor),
+            view.bottomAnchor.constraint(equalTo: tableView.bottomAnchor),
+        ]
+
+        NSLayoutConstraint.activate(constraints)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.tableHeaderView = headerView
+        tableView.tableFooterView = footerView
+
+        headerView.frame.size.height = 1
+        footerView.frame.size.height = 2
```

Comment:
Can we make those heights part of `AboutInstanceTableHeaderView` and `AboutInstanceTableFooterView` construction process rather than leaking this here? Or are they re-used and this height altered dynamically? In this case please disregard.